web-ui/src/components/certifications/Certifications.jsx

```diff
@@ -0,0 +1,254 @@
+import PropTypes from 'prop-types';
```

Comment:
This implements a dialog that is available to admin users to edit certifications.
"Earned certifications" are ones that have actually been earned by a member. They refer to a "certification".
"Certifications" are descriptions of possible certifications that can be shared by "earned certifications" ... such as a GCP certification.